### PR TITLE
Add history to VoucherApplication

### PR DESCRIPTION
**Pull request by hunytalk**
Add django-simple-history to VoucherApplication.
DE-1995

### CODE REVIEW

File: ecommerce/extensions/voucher/models.py [hunk 1]
```diff
@@ -6,7 +6,11 @@
 from django.core.exceptions import MultipleObjectsReturned, ObjectDoesNotExist
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
-from oscar.apps.voucher.abstract_models import AbstractVoucher  # pylint: disable=ungrouped-imports
+from oscar.apps.voucher.abstract_models import (  # pylint: disable=ungrouped-imports
+    AbstractVoucher,
```
Comment (HassanJaveed84) [inline ecommerce/extensions/voucher/models.py:10]: Do we need this ?

Comment (hunytalk) [reply ecommerce/extensions/voucher/models.py:10]: Do you mean AbstractVoucher?

Comment (hunytalk) [reply ecommerce/extensions/voucher/models.py:10]: Can you please explain further?

Comment (macdiesel) [reply ecommerce/extensions/voucher/models.py:10]: if you are talking about AbstractVoucher It's still used on line 36 with the Voucher class.

**Review by macdiesel — APPROVED** [review]

**Review by pwnage101 — APPROVED** [review]

**Pull request merged**